go.mod

```diff
@@ -1,69 +1,69 @@
 module mattermost-developer-documentation
 
-go 1.17
+go 1.18
 
 require (
-	github.com/mattermost/mattermost-server/v6 v6.3.0
+	// mmgoget: github.com/mattermost/mattermost-server/v6@v7.3.0 is replaced by -> github.com/mattermost/mattermost-server/v6@5d1fe271e6
+	github.com/mattermost/mattermost-server/v6 v6.0.0-20220914120725-5d1fe271e68c
```

Comment:
See https://community.mattermost.com/core/pl/xq1jy5pw8ibrmpx9uwswusozey for more context on why a commit is used instead of a version.